### PR TITLE
HDXDSYS-895 Split HAPI pipelines test into multiple tests

### PR DESCRIPTION
**Pull request by mcarans**

### CODE REVIEW

File: tests/__init__.py
Comment (turnerm) [inline tests/__init__.py:1]: Why is the contents below in `__init__.py`, could it go in to a utils file or something with a more obvious name?

**Review by turnerm — APPROVED** [review]
Really nice, the `test_main` file is so much more manageable now. 

I noticed that the tests take about 3 minutes more, I suppose that's expected but definitely worth it the trade-off. Could the tests be sped-up by limiting more of the sub-categories to a subset of countries?  I'm guessing that wouldn't help much.

**Review by b-j-mills — APPROVED** [review]
Looks great!  Agreed, so much easier to read and debug.

File: tests/__init__.py
Comment (mcarans) [reply tests/__init__.py:1]: I've done so

**Comment by mcarans** [comment]
@turnerm It could be worth timing each test and limiting subcategories for any slow ones.

**Pull request merged**